python-threatexchange/threatexchange/exchanges/clients/ncmec/hash_api.py

```diff
@@ -261,15 +343,19 @@ def __init__(
         username: str,
         password: str,
         environment: NCMECEnvironment,
+        member_id: t.Optional[str] = None,
+        reasons_map: t.Dict[str, t.List[t.Dict[str, str]]] = {},
     ) -> None:
         assert is_valid_user_pass(username, password)
         self.username = username
         self.password = password
         self._base_url = environment.value
+        self.member_id = member_id
+        self.reasons_map = reasons_map or {}
```

Comment:
Hmm this `or {}` is fairly subtle. I'm assuming we are doing it because the default is `{}` which is a shared instance across any object, and the `or {}` will replace it with a new instance per call, but having it be optional and default to  `None` is more easy to prove doesn't have a bug.